lib/features/dashboard/presentation/pages/dashboard_page.dart

```diff
@@ -0,0 +1,273 @@
+                        ),
+                      ),
+                    );
+                  },
+                ),
+              ),
+            ],
+          ),
+          const VerticalGap.xxxs(),
+             Row(
+            crossAxisAlignment: CrossAxisAlignment.start,
+            children: [
+              Expanded(
+                  child: BlocBuilder<HasItemCubit, HasItemState>(
+                  builder: (context, state) {
+                    return state.maybeWhen(
+                      loading: () => const DashboardCard(
+                        message: '',
+                        informationValue: 0,
+                        isLoading: true,
+                      ),
+                      error: () => DashboardCard(
+                        message: S.current.informational_card_error_message,
+                        onTap: () => context.read<HasItemCubit>().hasItemRegistered(),
+                      ),
+                      orElse: () => DashboardCard(
+                        message: S.current.dashboard_cheapest_item,
+                        informationValue: state.maybeWhen(
+                          orElse: () => 0,
+                          hasItemRegisters: _getLowestPrice,
+                        ),
+                      ),
+                    );
+                  },
+                ),
+              ),
+              const HorizontalGap.xxxs(),
+              Expanded(
+                child: BlocBuilder<HasItemCubit, HasItemState>(
+                  builder: (context, state) {
+                    return state.maybeWhen(
+                      loading: () => const DashboardCard(
+                        message: '',
+                        informationValue: 0,
+                        isLoading: true,
+                      ),
+                      error: () => DashboardCard(
+                        message: S.current.informational_card_error_message,
+                        onTap: () => context.read<HasItemCubit>().hasItemRegistered(),
+                      ),
+                      orElse: () => DashboardCard(
+                        message: S.current.dashboard_highest_item,
+                        informationValue: state.maybeWhen(
+                          orElse: () => 0,
+                          hasItemRegisters: _getHighestPrice,
+
+                        ),
+                      ),
+                    );
+                  },
+                ),
+              ),
+            ],
+          ),
+        ],
+      ),
+    );
+  }
+
+  List<Product> _getLowAmountItens(List<Product> products) {
+    List<Product> productsEnding = [];
+    for (final product in products) {
+  
+        productsEnding.add(product);
+    
+    }
+    return productsEnding;
+  }
+
+  int _getTotalPrice(List<Product> products) {
+    double totalPrice = 0;
+    for (final product in products) {
+      totalPrice += product.price;
+    }
+    return totalPrice.toInt();
+  }
```

Comment:
why to int if price is double? Return must be double.

Reply:
é pq a lista de produtos espera um inteiro